packages/foam-vscode/src/features/tags-tree-view/index.ts

```diff
@@ -102,23 +116,27 @@ type TagMetadata = { title: string; uri: URI };
 export class Tag extends vscode.TreeItem {
   constructor(
     public readonly tag: string,
+    public readonly title: string,
     public readonly notes: TagMetadata[]
   ) {
-    super(tag, vscode.TreeItemCollapsibleState.Collapsed);
+    super(title, vscode.TreeItemCollapsibleState.Collapsed);
     this.description = `${this.notes.length} reference${
       this.notes.length !== 1 ? 's' : ''
     }`;
     this.tooltip = getContainsTooltip(this.notes.map(n => n.title));
+    this.tag = tag;
```

Comment:
i don't believe you need this, as the `public readonly` in the argument `tag` is implicitly doing this assignment